bfvmm/CMakeLists.txt

```diff
@@ -49,6 +49,7 @@ add_subdirectory(src/memory_manager)
 add_subdirectory(src/debug)
 
 install(FILES include/vmm.h DESTINATION include/bfvmm)
+install(DIRECTORY include/interface DESTINATION include/bfvmm)
```

Comment:
@connojd Is there a better way that I should be installing a directory that contains nothing but header files? Do I need to turn it into an interface library?